packages/core/src/fields/types/image/views/Field.tsx

```diff
@@ -92,126 +90,167 @@ function ImgView({
+          <Stack
+            gap="small"
+            css={{
+              height: '120px',
+              justifyContent: 'space-between',
+            }}
+          >
+            {errorMessage === undefined ? (
+              <ImageMeta
+                {...(value.kind === 'from-server'
+                  ? {
+                      width: value.data.width,
+                      height: value.data.height,
+                      url: value.data.src,
+                      name: `${value.data.id}.${value.data.extension}`,
+                      size: value.data.filesize,
+                    }
+                  : {
+                      width: imageDimensions.width,
+                      height: imageDimensions.height,
+                      size: value.data.file.size,
+                    })}
+              />
+            ) : (
+              <Pill tone="negative" weight="light">
+                {errorMessage}
+              </Pill>
             )}
-            {value.kind === 'upload' && (
+            <Stack across gap="small" align="center">
               <Button
                 size="small"
-                tone="negative"
                 onClick={() => {
-                  onChange(value.previous);
+                  inputRef.current?.click();
                 }}
               >
-                Cancel
+                Change
               </Button>
-            )}
-            {errorMessage ? (
-              <Pill tone="negative" weight="light">
-                {errorMessage}
-              </Pill>
-            ) : (
-              value.kind === 'upload' && (
-                <Pill weight="light" tone="positive">
-                  Save to upload this image
-                </Pill>
-              )
-            )}
+              {value.kind === 'from-server' && (
+                <Button
+                  size="small"
+                  tone="negative"
+                  onClick={() => {
+                    onChange({ kind: 'remove', previous: value });
+                  }}
+                >
+                  Remove
+                </Button>
+              )}
+              {value.kind === 'upload' && (
+                <Button
+                  size="small"
+                  tone="negative"
+                  onClick={() => {
+                    onChange(value.previous);
+                  }}
+                >
+                  Cancel
+                </Button>
+              )}
+            </Stack>
           </Stack>
-        </Stack>
-      )}
-    </Stack>
-  ) : (
-    <Stack gap="small">
-      <Stack css={{ alignItems: 'center' }} gap="small" across>
-        <Button
-          size="small"
-          disabled={onChange === undefined}
-          onClick={() => {
-            inputRef.current?.click();
-          }}
-          tone="positive"
-        >
-          Upload Image
-        </Button>
-        {value.kind === 'remove' && value.previous && (
+        )
+      ) : (
+        <Fragment>
```

Comment:
Why the removal of a stack here?
<img width="260" alt="screenshot of Keystone Admin UI showing Upload Image and Undo Removal buttons with no spacing between them" src="https://user-images.githubusercontent.com/11481355/164371589-76135d89-4bc3-4005-bdf7-c3d8e00d81b4.png">

Reply:
Moving code is hard 😂, thanks for the catch 🎉